cirq/testing/random_circuit.py

```diff
@@ -37,13 +37,20 @@
 }  # type: Dict[ops.Gate, int]
 
 
-def random_circuit(qubits: Union[Sequence[ops.QubitId], int],
-                   n_moments: int,
-                   op_density: float,
-                   gate_domain: Optional[Dict[ops.Gate, int]] = None
-                   ) -> Circuit:
-    """Generates a random circuit.
+class RandomCircuit:
```

Comment:
I think that this should stay as a method. The `Type(args).type()` style is odd. Normally people would call it a factory or name the method `generate` or similar. I don't think we need to resort to factories yet.

Reply:
Deleting the class.